middleware/attemptSilentLogin.js

```diff
@@ -0,0 +1,36 @@
+const debug = require('../lib/debug');
+const COOKIES = require('../lib/cookies');
+
+const COOKIE_NAME = 'silentLoginAttempted';
+
+const cancelSilentLoginAttempts = (req, res) =>
+  res.cookie(COOKIE_NAME, true, {
+    httpOnly: true,
+    secure: req.secure,
```

Comment:
this ought to take into consideration an explicit configuration, similar to what we do in appSession.js

```
secure: typeof cookieConfig.secure === 'boolean'
          ? cookieConfig.secure
          : req.secure
```

Reply:
6c1cb89